### PR TITLE
Borgs and AIs now start with their borgHUDs/siliconHUDs on by default

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

see title

## Why It's Good For The Game

pushing this button at roundstart every round is mildly annoying, and many new silicon players don't even know that it exists

## Changelog
:cl: ATHATH
tweak: Borgs and AIs now start with their borgHUDs/siliconHUDs on by default.
/:cl:

### CODE REVIEW

**Comment by IndieanaJones** [comment]
Good change.

File: code/modules/mob/living/silicon/silicon.dm [hunk 1]
```diff
@@ -34,7 +34,7 @@
 	var/hackedcheck[1]
 	var/devillawcheck[5]
 
-	var/sensors_on = 0
+	var/sensors_on = 1 //are our HUDs on?
```
Comment (MrPerson) [inline code/modules/mob/living/silicon/silicon.dm:37]:
```suggestion
	var/sensors_on = TRUE //are our HUDs on?
```

**Comment by ExcessiveUseOfCobblestone** [comment]
https://github.com/tgstation/tgstation/pull/49808#issuecomment-596672475

I will accept this one since it is simple enough but please be aware that due to your history with the webbitor I will autoclose the PR if I see it has been webbited (Small Post-Initial PR changes are fine to webbit).

File: code/modules/mob/living/silicon/silicon.dm [hunk 2]
```diff
@@ -34,7 +34,7 @@
 	var/hackedcheck[1]
 	var/devillawcheck[5]
 
-	var/sensors_on = 0
+	var/sensors_on = TRUE //are our HUDs on?
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/silicon/silicon.dm:37]: do the autodoc

Comment (ATH1909) [reply code/modules/mob/living/silicon/silicon.dm:37]: ?

All of the other PRs I've seen just leave comments like this one; what should I be doing differently here?

Comment (zxaber) [reply code/modules/mob/living/silicon/silicon.dm:37]: It's for the [autodoc](http://codedocs.tgstation13.org/mob/living/silicon.html) page. All variables and procs are supposed to be documented using a specific format when they are declared. For example on a somewhat populated page, see the [mecha documentation](http://codedocs.tgstation13.org/obj/mecha.html).

Autodoc with three slashes before the var.

```
///Variable that holds the status of the silicon hud overlay. TRUE for enabled, FALSE for disabled.
var/sensors_on = TRUE

**Comment by ATH1909** [comment]
To be clear, I'm considering complying with our autodoc requirements to be a small, post-initial PR change.

**Pull request merged**